phabfive/paste.py

```diff
@@ -32,6 +35,46 @@ def _convert_ids(self, ids):
 
         return ids_list_int
 
+    def create_paste(
+        self, title=None, file=None, language=None, tags=None, subscribers=None
+    ):
+        """Wrapper that connects to Phabricator and creates paste.
+
+        :type title: str
+        :type file: str
+        :type language: str
+        :type tags: list
+        :type subscribers: list
+
+        :rtype: dict
+        """
+        text = None
+        tags = tags if tags else []
+        subscribers = subscribers if subscribers else []
+
+        with open(file, "r") as f:
+            text = f.read()
+
+        transactions = []
+        transactions_values = [
+            {"type": "title", "value": title},
+            {"type": "text", "value": text},
+            {"type": "language", "value": language},
+            # TODO: Create a function that vaildates tags' and 'subscribers' existence
+            {"type": "projects.add", "value": tags},
+            {"type": "subscribers.add", "value": subscribers},
+        ]
+        # Phabricator does not take None (empty list is ok for projects/subscribers) as a value, therefor only "type" that has valid value can be sent as an argument
+        for item in transactions_values:
+            if None not in item.values():
+                transactions.append(item)
```

Comment:
Can be simplified to list comprehension `transactions = [item for item in transaction_values if None not in item.values()]`